src/main/jbake/content/blog/2016/09/06/consuming-hawkular-api-over-ssl.adoc

```diff
@@ -0,0 +1,84 @@
+= Consuming Hawkular API over SSL with self signed certificates
+Josejulio Martínez
+2016-9-6
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+In this post I'll show you how to trust self-signed certificates for use with Ruby client, used on projects such as ManageIQ and HawkFX.
+
+If you haven't already, follow this guide for
+link:http://www.hawkular.org/hawkular-services/docs/installation-guide/secure-comm.html[securing communications]
+ on your Hawkular server, because we'll make use of the files generated by that guide.
+
+As today, there are Hawkular clients for the following programming languages: Java, Python, Go and link:http://github.com/hawkular/hawkular-client-ruby[Ruby].
+The Hawkular Ruby client lets you connect using a secure connection. Buying a certificate from a trusted SSL vendor to host your
+own development server can be cost-prohibitive or just meaningless.
```

Comment:
I would use a more positive word, `meaningless` is very negative. Maybe `complicated` ...